packages/lesswrong/components/peopleDirectory/PeopleDirectoryFilters.tsx

```diff
@@ -69,21 +75,49 @@ export const PeopleDirectoryFilters = ({classes}: {
       </div>
       <div className={classes.options}>
         {view === "list" &&
-          <PeopleDirectoryFilterDropdown
-            title="Columns"
-            icon="ViewColumns"
-            style="button"
-            className={classes.columnsList}
-          >
-            {columns.filter(({hideable}) => hideable).map((state) => (
-              <PeopleDirectorySelectOption state={state} key={state.value} />
-            ))}
-            {columnsEdited &&
-              <div>
-                <PeopleDirectoryClearAll text="Reset" onClear={resetColumns} />
-              </div>
-            }
-          </PeopleDirectoryFilterDropdown>
+          <>
+            <PeopleDirectoryFilterDropdown
+              title="Sort"
+              icon="ArrowsUpDown"
+              style="button"
+              className={classes.columnsList}
+            >
+              <PeopleDirectoryCheckOption
+                label="Default"
+                selected={isDefaultSorting}
+                onSelect={setSorting.bind(null, null)}
+              />
+              {columns.map(({label, sortField, defaultSort}) => sortField
```

Comment:
It's a bit confusing that there is no column for "Location" or "Profile last updated" when you sort by them. I think if it added the column of the thing you are sorting by that would be a lot better (although still a bit confusing)

The comments sorting is also a quite confusing (and also didn't work for me due to an elastic error "No mapping found for [commentCount] in order to sort on"). I would favour either showing the number of comments or just not having that sorting if there's a reason we don't want to do that